group/assignment6/tests.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+from typing import List, Tuple
+from move_cars import *
+
+def check_movement_correctness(start: List[int], final: List[int], \
+                               moves: List[Tuple[int]]) -> bool:
+    # each move has format (car number, old space, new space)
+    current_pos = start[:]
+    for move in moves:
+        car, old_space, new_space = move
+        current_pos[old_space] = 0
```

Comment:
I think we want to check that the new space had a zero here
And that old_space had car
And swap